includes/results.inc

```diff
@@ -847,8 +847,8 @@ class IslandoraSolrFacets {
     foreach ($results as $bucket => $count) {
       $facet_results[] = array(
         'count' => $count,
-        'filter' => islandora_solr_lesser_escape($facet_field) . ':"' .
-        islandora_solr_lesser_escape($bucket) . '"',
+        'filter' => islandora_solr_facet_escape($facet_field) . ':"' .
```

Comment:
So did you guys work this out? I agree with @adam-vessey, this is not required as your "1/4" tape" is a $facet_value the $facet_field is mods_physicalDescription_form_s.

But it also shouldn't do any damage, so I'm leaving it to the pros.